### DISCUSSION TITLE
❔ How to Contribute / Which Issues to Select?

### DISCUSSION THREAD

**Discussion by EmperorOrokuSaki** [pcaversaccio/snekmate#226] [Q&A] [answered]
Hi! When going through the repository and the issues I noticed all of them are already assigned and presumably being worked on.
I'd like to pick up some of them and start contributing, however, it's unclear what I can start with.
Edit: I have noticed the `help wanted` label on some, but wanted to confirm that they're still active issues.
cc @pcaversaccio

**Comment by pcaversaccio** [comment, marked as answer]
> Hi! When going through the repository and the issues I noticed all of them are already assigned and presumably being worked on. I'd like to pick up some of them and start contributing, however, it's unclear what I can start with. Edit: I have noticed the `help wanted` label on some, but wanted to confirm that they're still active issues. cc @pcaversaccio

Hey @EmperorOrokuSaki, appreciate your willingness to help. I will quickly refer to my comment [here](https://github.com/pcaversaccio/snekmate/issues/176#issuecomment-2031612096):

> Thanks for the ping. So the situation is the following: Vyper will release soonish a new breaking version (see the beta version [here](https://github.com/vyperlang/vyper/releases/tag/v0.4.0b1)) that allows for stateless and stateful modules. I'm in the process of refactoring the 🐍 snekmate contracts for this release (you can follow the process in this PR: https://github.com/pcaversaccio/snekmate/pull/207). These new language features will power a lot of the features in the open issues I have in mind. But until Vyper `0.4.0` is shipped and I have not refactored all contracts, there is an ongoing feature freeze. So it could be a few weeks before we can start with any new issue.

Can I ping you here once we can move on with new features?

**Reply by EmperorOrokuSaki** [reply]
Perfect. Thanks for the quick response! I'll keep an eye out for the release and touch base again after that.
